ext/openssl/openssl.stub.php

```diff
@@ -203,32 +195,32 @@ function openssl_get_cipher_methods(bool $aliases = false): array {}
 function openssl_get_curve_names(): array|false {}
 #endif
 
-function openssl_digest(string $data, string $method, bool $raw_output = false): string|false {}
+function openssl_digest(string $data, string $digest_algorithm, bool $raw_output = false): string|false {}
 
 /** @param string $tag */
-function openssl_encrypt(string $data, string $method, string $password, int $options = 0, string $iv = '', &$tag = UNKNOWN, string $aad = '', int $tag_length = 16): string|false {}
+function openssl_encrypt(string $data, string $cipher_algorithm, string $passphrase, int $options = 0, string $initialization_vector = "", &$tag = null, string $additional_authentication_data = "", int $tag_length = 16): string|false {}
 
-function openssl_decrypt(string $data, string $method, string $password, int $options = 0, string $iv = '', string $tag = UNKNOWN, string $aad = ''): string|false {}
+function openssl_decrypt(string $data, string $cipher_algorithm, string $passphrase, int $options = 0, string $initialization_vector = "", ?string $tag = null, string $additional_authentication_data = ""): string|false {}
 
-function openssl_cipher_iv_length(string $method): int|false {}
+function openssl_cipher_iv_length(string $cipher_algorithm): int|false {}
 
-function openssl_dh_compute_key(string $pub_key, OpenSSLAsymmetricKey $dh_key): string|false {}
+function openssl_dh_compute_key(string $public_key, OpenSSLAsymmetricKey $private_key): string|false {}
 
 /**
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $peer_pub_key
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $priv_key
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $public_key
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key
  */
-function openssl_pkey_derive($peer_pub_key, $priv_key, int $keylen = 0): string|false {}
+function openssl_pkey_derive($public_key, $private_key, int $key_length = 0): string|false {}
 
-/** @param bool $result_is_strong */
-function openssl_random_pseudo_bytes(int $length, &$result_is_strong = UNKNOWN): string {}
+/** @param bool $strong_result */
+function openssl_random_pseudo_bytes(int $length, &$strong_result = null): string {}
 
-function openssl_spki_new(OpenSSLAsymmetricKey $privkey, string $challenge, int $algo = OPENSSL_ALGO_MD5): string|false {}
+function openssl_spki_new(OpenSSLAsymmetricKey $private_key, string $challenge, int $digest_algorithm = OPENSSL_ALGO_MD5): string|false {}
 
-function openssl_spki_verify(string $spki): bool {}
+function openssl_spki_verify(string $signed_public_key_and_challenge): bool {}
```

Comment:
```suggestion
function openssl_spki_verify(string $spki): bool {}
```
The functions are called openssl_spki_*, so I think using that is more appropriate...

Reply:
Agreed! $signed_public_key_and_challenge` seemed too verbose for me as well